package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "voxa",
-  "version": "3.0.0",
+  "version": "3.0.0-alpha43",
```

Comment:
please remove this

Reply:
Hmm why? I'm planning to release this to NPM to the alpha channel

Reply:
So, once we released 3.0.0 the next alpha should be 3.0.1-alpha or 3.1.0-alpha depending on semver. Seems like this would 3.1.0 because there's a new feature

Reply:
Got it, thanks!